include/alpaka/mem/buf/BufCpu.hpp

```diff
@@ -55,7 +55,7 @@ namespace alpaka
                 DevCpu dev,
                 TElem* pMem,
                 std::function<void(TElem*)> deleter,
-                TExtent const& extent)
+                TExtent const& extent) noexcept
```

Comment:
Technically, the constructor uses `std::cout << ...`, which is not `noexcept`. I don't expect it to throw though :) If that places g++-12 so be it.

Reply:
Well, we only `std::cout` stuff when we're in debug mode. When we're in debug mode and `cout` throws we probably have a much worse problem at hand ;-)